chucks-pick3-web/angular-app/src/app/components/info-interface.component/info-interface.component.html

```diff
@@ -1,10 +1,10 @@
 <div class="how-to-header" id="interface">
   <h1><b>Interface:</b></h1>
-  <img src="/assets/images/Info page -04.png" alt="Interface" style="width:700px" align="middle">
+  <img src="src/app/assets/images/Info page -04.png" alt="Interface" style="width:700px" align="middle">
```

Comment:
Not Sure about this change.  Does this still work?  I think it is okay to start paths to images and css files with just `assets/images` and `assets/css`;  Please revert back.

Reply:
reverted back to assets/images.